_posts/2019-08-12-how-to-configure-production-grade-aws-account-structure.adoc

```diff
@@ -2063,16 +2196,6 @@ The example above configures the logs account of an AWS Organization as follows:
 . **Allow access from the security account**: We configure IAM roles that IAM users in the security account will be
   able to assume to get access to the logs account.
 
-Configure your Terraform backend:
-
-.infrastructure-live/logs/_global/account-baseline/terragrunt.hcl
-[source,hcl]
-----
-include {
```

Comment:
Don't you need this in every child `terragrunt.hcl`?

Reply:
Or did you move this snippet later in the guide for some reason?

Reply:
Yes, I thought it made more sense to order it this way. The way our generated files look, this is where the `include` block appears. Does it make less sense now?